feature-libs/cart/import-export/components/import-to-cart/import-to-cart.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { ImportToCartService } from './import-to-cart.service';
+import { ImportExportConfig } from '@spartacus/cart/import-export/core';
+
+@Component({
+  selector: 'cx-import-to-cart',
+  templateUrl: './import-to-cart.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ImportToCartComponent {
```

Comment:
I know that import is always to cart, but maybe is worth to consider the name as `ImportEntriesComponent` for consistency with export? This component is in `cart` folder itself.